control_plane/src/background_process.rs

```diff
@@ -0,0 +1,250 @@
+//! Spawns and kills background processes that are needed by Neon CLI.
+//! Applies common set-up such as log and pid files to every process.
```

Comment:
pid file is created by the process itself, not this.

Reply:
Reworded slightly, since etcd still forces us to create one.